CIP-0043/README.md

```diff
@@ -78,51 +79,62 @@ functions, these are:
 2. An input to verify (either the message itself, or a hash);
 3. A signature.
 
-The two different schemes have different expectations of each argument and how
-it is encoded into a `BuiltinByteString`. We describe these below.
+The two different schemes handle deserialization internally: specifically, there
+is a distinction made between 'external' representations, which are expected as
+arguments, and 'internal' representations, used only by the implementations
+themselves. This creates different expecations for each argument for both of
+these schemes; we describe these below.
 
-For the ECDSA signature scheme, we have the following requirements:
+For the [ECDSA signature
+scheme](https://en.bitcoin.it/wiki/Elliptic_Curve_Digital_Signature_Algorithm),
+the requirements are as follows. Note that these differ from the serialization
+used by Bitcoin.
 
+* The verification key must correspond to the _(x, y)_ coordinates of a point 
+  on the SECP256k1 curve, where _x, y_ are unsigned integers in big-endian form.
 * The verification key must correspond to a result produced by
   [``secp256k1_ec_pubkey_serialize``](https://github.com/bitcoin-core/secp256k1/blob/master/include/secp256k1.h#L394), 
   when given a length argument of 33, and the ``SECP256K1_EC_COMPRESSED`` flag.
   This implies all of the following:
     * The verification key is 33 bytes long.
-    * The first byte corresponds to the sign of the _y_ coordinate; this is
+    * The first byte corresponds to the parity of the _y_ coordinate; this is
       `0x02` if _y_ is even, and `0x03` otherwise.
-    * The remaining 32 bytes are the bytes of the _x_ coordinate, as a
-      big-endian integer.
+    * The remaining 32 bytes are the bytes of the _x_ coordinate.
 * The input to verify must be a 32-byte hash of the message to be checked. We 
   assume that the caller of `verifyEcdsaSecp256k1Signature` receives the 
   message and hashes it, rather than accepting a hash directly: doing so 
   [can be dangerous](https://bitcoin.stackexchange.com/a/81116/35586).
   Typically, the hashing function used would be SHA256; however, this is not
   required, as only the length is checked.
+* The signature must correspond to two unsigned integers in big-endian form;
```

Comment:
Same as above. In this case it is a bit less confusing as both formats match, but maybe I would again rephrase in that the first bullet point is what the signature is, and the second is the compressed format accepted by the builtins.